### PR TITLE
Fix a minor style issue in nn_ops and math_ops

### PR DESCRIPTION
**Pull request by autoih**

### CODE REVIEW

File: tensorflow/python/ops/math_ops.py [hunk 1]
```diff
@@ -4104,7 +4104,8 @@ def _tensordot_axes(a, axes):
       a_free_dims = ops.convert_to_tensor(a_free_dims, dtype=dtypes.int32)
       b_free_dims = ops.convert_to_tensor(b_free_dims, dtype=dtypes.int32)
       product = array_ops.reshape(
-          ab_matmul, array_ops.concat([a_free_dims, b_free_dims], 0), name=name)
+          ab_matmul, array_ops.concat(
+          [a_free_dims, b_free_dims], 0), name=name)
```
Comment (tomhennigan) [inline tensorflow/python/ops/math_ops.py]: I'm not sure that this actually provides a more readable result?

I'd suggest reverting to the before:

```
product = array_ops.reshape(
    ab_matmul, array_ops.concat([a_free_dims, b_free_dims], 0), name=name)
```

Or to keep every parameter to the function on it's own line:

```
product = array_ops.reshape(ab_matmul,
                            array_ops.concat([a_free_dims, b_free_dims], 0),
                            name=name)
```

**Review by tomhennigan — COMMENTED** [review]
Thanks for the PR! I think some of the doc fixes are in line with PEP-8, but actually show an interesting case where PEP-8 [0] and the Google Python Style Guide [1] disagree (79 vs 80ch lines). TF style guide is to follow both [2].

@martinwicke any guidance on which rule should "win" here?

[0] https://www.python.org/dev/peps/pep-0008/#maximum-line-length
[1] https://github.com/google/styleguide/blob/gh-pages/pyguide.md#32-line-length
[2] https://www.tensorflow.org/community/contribute/code_style

File: tensorflow/python/ops/math_ops.py (hunk 1 above)
Comment (autoih) [reply tensorflow/python/ops/math_ops.py]: Thanks @tomhennigan. The both two ways exceed 80 char in a line. Which one do you prefer to edit?

Comment (tomhennigan) [reply tensorflow/python/ops/math_ops.py]: I _think_ that the if we leave it as is (e.g. the first suggestion) it's exactly 80ch so I'd suggest leaving it as is for now:

```shell
$ echo -n "          ab_matmul, array_ops.concat([a_free_dims, b_free_dims], 0), name=name)" | wc -c
      80
```

**Review by tomhennigan — DISMISSED** [review]

**Comment by gbaned** [comment]
@autoih Could you please address Ubuntu Sanity errors? Thanks!

**Comment by autoih** [comment]
Thanks @gbaned.  Since I cannot find the log message, can you help me trigger it again?

**Comment by gbaned** [comment]
@autoih Can you please check it now. Thanks!

**Review by alextp — APPROVED** [review]

**Pull request merged**